README.md

```diff
@@ -233,10 +233,10 @@ twig:
             - SimpleThingsFormExtraBundle:Form:field_type_help.html.twig
 ```
 
-You can also load the help extension form theme into your own form theme.
+Next, load the help extension form theme into your template.
```

Comment:
That actually made it work for me. The previous instructions changed nothing.
Using Symfony 2.8

Are you sure what I added is not the proper way?